internal/expr/query.go

```diff
@@ -62,7 +62,7 @@ func (pe *PreparedExpr) Query(args ...any) (ce *QueryExpr, err error) {
 			// Check if we have a type with the same name from a different package.
 			for _, typeMember := range pe.inputs {
 				if t.Name() == typeMember.outerType().Name() {
-					return nil, fmt.Errorf("type %s not passed as a parameter, have %s", typeMember.outerType().String(), t.String())
+					return nil, fmt.Errorf("type %q not found, have type with same name: %q", typeMember.outerType().String(), t.String())
```

Comment:
If that's the exact same issue, the message should also be the same?

Reply:
It was changed because of my comment above https://github.com/canonical/sqlair/pull/105#discussion_r1335522636, the problem is that when the type name is the same it is not clear in the error message why it is not working.

Reply:
I've changed it to the new message that we use in `typeMissingError`.